target/idl/uxd.json

```diff
@@ -3398,136 +3436,141 @@
     },
     {
       "code": 6030,
```

Comment:
For this reason I think we should append error at the end of the file (so that error codes keep the same meaning for the future when someone look in the past tx onchain)

Reply:
Smart, ok will do 👌